addons/hr/static/src/components/avatar_card/avatar_card_popover_patch.js

```diff
@@ -17,7 +17,7 @@ export const patchAvatarCardPopover = {
             "work_email", 
             "job_title", 
             "department_id", 
-            "employee_ids",
+            this.props.recordModel ? 'employee_id' : 'employee_ids',
```

Comment:
This is the formatting convention used by the odoo js linter:
```suggestion
            this.props.recordModel ? "employee_id" : "employee_ids",
```